modules/imdsBidAdapter.js

```diff
@@ -38,21 +39,23 @@ export const spec = {
       return;
     }
     const refererInfo = bidderRequest.refererInfo;
-    const openRtbBidRequest = {
+    // start with some defaults, overridden by anything set in ortb2, if provided.
+    const openRtbBidRequest = mergeDeep({
       id: bidderRequest.bidderRequestId,
       site: {
-        // TODO: does the fallback make sense here?
-        domain: refererInfo.domain || location.hostname,
+        domain: refererInfo.domain,
         page: refererInfo.page,
         ref: refererInfo.ref
       },
       device: {
         ua: navigator.userAgent
       },
       imp: []
-    };
+    }, bidderRequest.ortb2 || {});
 
-    const tmax = bidderRequest.timeout;
+    const callbackTimeout = bidderRequest.timeout;
+    const globalTimeout = config.getConfig('bidderTimeout');
+    const tmax = globalTimeout ? Math.min(globalTimeout, callbackTimeout) : callbackTimeout;
```

Comment:
this is incorrect, just use the value in the request

Reply:
> this is incorrect, just use the value in the request

Okay, I was wondering how https://github.com/prebid/Prebid.js/pull/8526 somehow disappeared.